dbms/src/Storages/DeltaMerge/DeltaTree.h

```diff
@@ -174,8 +174,8 @@ struct DTLeaf
 
     static inline bool overflow(size_t count) { return count > M * S; }
     static inline bool underflow(size_t count) { return count < M; }
-    inline bool legal() { return !overflow(count) && !underflow(count); }
```

Comment:
Are the changes in this file related to the issue that this PR is trying to fix?

Reply:
No, reverted